horizon/facts/router.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, Depends, Request as FastApiRequest
+from loguru import logger
+
+from authentication import enforce_pdp_token
+from facts.client import FactsClientDependency
+
+facts_router = APIRouter(dependencies=[Depends(enforce_pdp_token)])
+
+
+@facts_router.post("/users")
+async def create_user(request: FastApiRequest, client: FactsClientDependency):
```

Comment:
I guess that you'll implement the remaining once you finish to fully implement one ?